cookbooks/Berksfile.lock

```diff
@@ -29,6 +29,8 @@ DEPENDENCIES
     path: cdo-secrets
   cdo-solr
     path: cdo-solr
+  cdo-sqs
+    path: cdo-sqs
```

Comment:
I don't see this cookbook anywhere in this PR or on staging, is this supposed to be in this PR or are these entries supposed to be removed?